ast/builtins.go

```diff
@@ -1721,6 +1722,24 @@ var AddDate = &Builtin{
 	),
 }
 
+// Diff returns the difference [years, months, days, hours, minutes, seconds] between two unix timestamps in nanoseconds
+var Diff = &Builtin{
+	Name: "time.diff",
+	Decl: types.NewFunction(
```

Comment:
🤔 is this type correct? I don't see where we'd allow for strings in any of the arguments. And the return value is more than 3 numbers.

Reply:
The string argument comes from when the option of specifying the timezone, similar to  `time.clock`, `time.date` and `time.weekday`. Default is "UTC". See https://github.com/open-policy-agent/opa/pull/3358/files#diff-635e7ef6d0974b01a1e80c081fbdb4c5b8d03e6787f3c25a28d8038112792adfR778.

As for the return value, you are right. I will add 2 more numbers

Reply:
Ah I wasn't aware of the tz feature 😅 -- great, thanks.